tests_e2e/test_e2e.py

```diff
@@ -51,7 +51,10 @@ def delete_trello_board(id):
 
 @pytest.fixture(scope='module')
 def driver():
-    with webdriver.Firefox() as driver:
+    opts = webdriver.ChromeOptions()
+    opts.add_argument('--headless')
+    opts.add_argument('--no-sandbox')
+    with webdriver.Chrome('./chromedriver', options=opts) as driver:
```

Comment:
I couldn't find a way to add chromedriver to my path in my Dockerfile and have it persist until the point when the tests run... Could we maybe have a chat about how this would be achieved please @JackMead?

I found a different solution, which was to install chromedriver on the fly using `ChromeDriverManager().install()` (part of the `webdriver_manager` package). Interested to know whether this is better or worse for any reason!